lib/membrane/aac/parser/helper.ex

```diff
@@ -144,9 +145,9 @@ defmodule Membrane.AAC.Parser.Helper do
       # home
       0::1,
       # copyright identification bit
-      1::1,
+      0::1,
       # copyright identification start
-      1::1,
+      0::1,
```

Comment:
I am not sure about this one. Is this correct to create aac with adts where there is no copyright identification bit set to 1 in the whole stream?

Reply:
FFmpeg apparently does that (https://github.com/FFmpeg/FFmpeg/blob/877ccaf776c92866e17493b48da49fab5b9ef11a/libavformat/adtsenc.c#L157) so IMO that's good enough :wink:

Reply:
Great, so IMO after improving raise message we are ready to merge